PLIER_pathways_analysis.Rmd

```diff
@@ -0,0 +1,76 @@
+---
+title: "PLIER pathways analysis"
+output: html_notebook
+---
+
+```{r}
+meaningful_difference <- 0.2
+```
+
+### What additional pathways are identified by PLIER after doubling the sample size?
+
+Here we look for cancer pathways that are found more frequently in the data with the full sample size than in data with half the sample size.
+We ran PLIER 10 times using each combination of data normalization method and sample size (half or full).
+Out of those 10 runs, we found the proportion of times each pathway was significantly associated with a latent variable.
```

Comment:
If I understand correctly, we output _all_ the pathways in `7-extract_plier_pathways.R` but we're only going to display the proportion information for `oncogenicPathways` below.

```suggestion
Out of those 10 runs, we found the proportion of times each oncogenic pathway was significantly associated with a latent variable.
```

Reply:
Yes good to specify, and I changed the order of events in the piping to promote this filtering step